packages/identity-service/src/notifications/emails/otp.js

```diff
@@ -1,362 +1,776 @@
-    <xml>
-    <o:OfficeDocumentSettings>
-    <o:AllowPNG/>
-    <o:PixelsPerInch>96</o:PixelsPerInch>
-    </o:OfficeDocumentSettings>
-    </xml>
-    <![endif]--> 
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"> 
-    <meta name="viewport" content="width=device-width,initial-scale=1"> 
-    <meta http-equiv="X-UA-Compatible" content="IE=Edge"/> 
-    <meta name="x-apple-disable-message-reformatting"> 
-    <meta name="color-scheme" content="light only">
-    <title></title> 
-    <style>html{-webkit-text-size-adjust:none;-ms-text-size-adjust:none}@media only screen and (max-device-width:600px),only screen and (max-width:600px){.mob_100{width:100%!important;max-width:100%!important}.mob_full{width:auto!important;display:block!important;padding:0 10px!important}.mob_center{text-align:center!important}.mob_center_bl{margin-left:auto;margin-right:auto}.mob_hidden{display:none!important}.only_mob{display:block!important}}@media only screen and (max-width:600px){.mob_100{width:100%!important;max-width:100%!important}.mob_100 img,.mob_100 table{max-width:100%!important}.mob_full{width:auto!important;display:block!important;padding:0 10px!important}.mob_center{text-align:center!important}.mob_center_bl{margin-left:auto;margin-right:auto}.mob_hidden{display:none!important}.only_mob{display:block!important}}.creative{width:100%!important;max-width:100%!important}.mail_preheader{display:none!important}form input, form textarea{font-family: Arial, sans-serif;width: 100%;box-sizing: border-box;font-size: 13px;color:#000000;outline:none;padding: 0px 15px;}form textarea{resize:vertical;line-height: normal;padding: 10px 15px;}form button{border: 0px none;cursor:pointer;}</style> 
-    <style>@media only screen and (max-width:480px){u+.body .full-wrap{width:100%!important;width:100vw!important}}</style> 
-    <style>@-ms-viewport{width:device-width}</style> 
-    <!--[if (gte mso 9)|(IE)]>
-    <style type="text/css">table {border-collapse: collapse !important;}.outf14{font-size:14px !important;}    .w520px{ width: 520px !important;} .w520px{ width: 520px !important;} .not_for_outlook{mso-hide: all !important;display: none !important;font-size:0;max-height:0;line-height: 0;mso-hide: all;}.outpadding{padding-left: 0 !important;padding-right: 0 !important;}</style>
-    <![endif]--> 
-    </head> 
-    <body class="body" style="padding:0;margin:0"> 
-    <div class="full-wrap"> 
-    <table width="100%" border="0" cellspacing="0" cellpadding="0" class="full-wrap">
-    <tr><td align="center" bgcolor="#f5f5f5" style="line-height: normal; hyphens: none;">
-    <div>
-    <!--[if !mso]>
-    <!-->
-    <div class="mail_preheader" style="font-size: 0px; color: transparent; opacity: 0;">
-      <span style="font-family: Arial, Helvetica, sans-serif; font-size: 0px; color: transparent; line-height: 0px;"></span>
-    </div>
-    <!--
-    <![endif]-->
-  </div>  
+const getOtpEmail = ({ otp, expire, copyrightYear }) => {
```

Comment:
How plug-n-play was this from Figma? Did you just have to interpolate the values, and the rest was copy pasta?

Reply:
mostly copy and paste, some small things we need to look out for in general like the copyright year, some links should open in a new tab, some inline links are missing in the template